client/notices/docs/example.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import ButtonGroup from 'components/button-group';
+import i18n from 'lib/mixins/i18n';
+import notices from 'notices';
+
+const NoticesGlobal = () => {
+	function callSuccessNotice() {
```

Comment:
Super minor, but these should live in the module scope so that we don't create these functions on every render.